docs/admin/deployment/landing_page.rst

```diff
@@ -140,10 +140,11 @@ let us know and we can remove your instance from the directory.
 URL and Location
 ----------------
 
-Ideally you would not use a separate subdomain, but would use a path at
-your top-level domain, e.g. organization.com/securedrop. This is because
-TLS does not encrypt the hostname, so a SecureDrop user whose connection
-is being monitored would be trivially discovered.
+Your *Landing Page* must be a path at your top-level domain, e.g. 
+organization.com/securedrop, rather than a subdomain (e.g., 
+securedrop.organization.com). This is because TLS does not encrypt the hostname,
```

Comment:
```suggestion
securedrop.organization.com). This is because DNS and TLS do not always encrypt the hostname,
```

ECH (https://en.wikipedia.org/wiki/Server_Name_Indication#Encrypted_Client_Hello) has started rollout, so it's no longer an absolute if people do have DoH/DoT enabled